bgpd/bgp_label.h

```diff
@@ -94,4 +94,21 @@ static inline uint8_t label_bos(mpls_label_t *label)
 	return (t[2] & 0x01);
 };
 
+static inline bool
+bgp_path_has_valid_label_const(const struct bgp_path_info *path)
+{
+	if (!path->extra)
+		return false;
+	if (!path->extra->num_labels)
+		return false;
+
+	return bgp_is_valid_label(&path->extra->label[0]);
+}
+
+static inline bool bgp_path_has_valid_label(struct bgp_path_info *path)
+{
+	return bgp_path_has_valid_label_const(
```

Comment:
Why do we need two functions const vs. non-const?

Reply:
Only one const function now: bgp_path_has_valid_label()